### PR TITLE
[docfx] remove _op_gitCommitHistory in op.html

### PR DESCRIPTION
**Pull request by herohua**
@vwxyzh @superyyrrzz @vicancy

### CODE REVIEW

File: src/docfx.website.themes/op.html/conceptual.mta.json.js [hunk 1]
```diff
@@ -12,7 +12,8 @@ function transform(model, _attrs){
   model._op_clientId = undefined;
   model._op_clientSecret = undefined;
   model._op_gitContributorInformation = undefined;
-  model.newFileRepository = undefined;
```
Comment (vicancy) [inline src/docfx.website.themes/op.html/conceptual.mta.json.js]: why removing this line?

Comment (herohua) [inline src/docfx.website.themes/op.html/conceptual.mta.json.js]: nice catch, should keep

File: src/docfx.website.themes/op.html/conceptual.mta.json.js [hunk 2]
```diff
@@ -12,6 +12,8 @@ function transform(model, _attrs){
   model._op_clientId = undefined;
   model._op_clientSecret = undefined;
   model._op_gitContributorInformation = undefined;
+  model._op_gitContributorInformation = undefined;
```
Comment (vicancy) [inline src/docfx.website.themes/op.html/conceptual.mta.json.js]: duplicate with L14?

**Comment by vicancy** [comment]
LGTM

**Comment by herohua** [comment]
Checked-in. Thanks!

**Pull request closed without merging**